<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamFeed</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="RecStopBtn">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="RightCamImg">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LeftCamImg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="LeftCamCropSlider">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="RightCamCropSlider">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="RightSliderValLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>31</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="LeftSliderValLabel">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>590</y>
      <width>31</width>
      <height>39</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Left Camera cropping"/>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>RightSliderValLabel</zorder>
   <zorder>LeftSliderValLabel</zorder>
   <zorder>LeftCamImg</zorder>
   <zorder>RightCamImg</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LeftCamCropSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LeftSliderValLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RightCamCropSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RightSliderValLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
